Source/Serilog.Exceptions/Destructurers/ReflectionTypeLoadExceptionDestructurer.cs

```diff
@@ -28,9 +27,29 @@ public class ReflectionTypeLoadExceptionDestructurer : ExceptionDestructurer
             {
                 propertiesBag.AddProperty(
                     nameof(ReflectionTypeLoadException.LoaderExceptions),
-                    reflectionTypeLoadException.LoaderExceptions.Select(destructureException).ToList());
+                    GetLoaderExceptionsValue(reflectionTypeLoadException.LoaderExceptions, destructureException));

```

Comment:
Code changed here to stop using Linq and fix potential null reference exceptions.